com.roundabout_host.Izvor.yaml

```diff
@@ -0,0 +1,70 @@
+app-id: com.roundabout_host.Izvor
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: izvor
+finish-args:
+  - --share=network
+  - --filesystem=home
+  - --filesystem=host
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=wayland
+  - --filesystem=xdg-data/izvor:create
+  - --filesystem=xdg-config/izvor:create
+environment:
+  PYTHONPATH: "/app/lib/python3.12/site-packages:${PYTHONPATH}"
+modules:
+  - name: python-requirements
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/python_packages
+      - cp -r . /app/share/python_packages/
+    sources:
+      - type: dir
+        path: python_packages
+  - name: izvor
+    buildsystem: simple
+    build-commands:
+      - unset GTK_MODULES
+      - mkdir -p /app/bin /app/share/izvor
+      - cp -r ./* /app/share/izvor/
+      - ls /app/share/python_packages
+      - pip install --no-index --find-links=/app/share/python_packages -r /app/share/izvor/requirements-flatpak.txt --target=/app/lib/python3.12/site-packages
+      - chmod +x /app/share/izvor/__init__.py
+      - ln -s /app/share/izvor/__init__.py /app/bin/izvor
+      #- fc-cache -fv
+    sources:
+      - type: dir
```

Comment:
1. don't use `dir` source
2. there is no `src` so I doubt this build was even tried because this is a fatal error.